publication-commons/src/test/java/no/unit/nva/publication/service/impl/DynamoDBPublicationServiceTest.java

```diff
@@ -46,7 +59,7 @@
     @Rule
```

Comment:
Are we sure this is working with Junit5?

Reply:
Yes, because of @EnableRuleMigrationSupport